_topic_map.yml

```diff
@@ -510,6 +489,28 @@ Topics:
   File: revhistory_install_config
   Distros: openshift-enterprise
 ---
+Name: Upgrading Clusters
+Dir: upgrading
+Distros: openshift-origin,openshift-enterprise
+Topics:
+- Name: Upgrade Methods and Strategies
+  File: index
+- Name: Automated In-place Upgrades
+  File: automated_upgrades
+- Name: Manual In-place Upgrades
+  File: manual_upgrades
+- Name: Blue-Green Deployments
+  File: blue_green_deployments
+- Name: Operating System Updates and Upgrades
+  File: os_upgrades
+- Name: Migrating Embedded etcd to External etcd
+  File: migrating_embedded_etcd
+- Name: Migrating etcd Data (v2 to v3)
```

Comment:
Checking if this v2 to v3 topic should stick around or be retired

Reply:
This is about etcd v2 to v3 and not OpenShift overall, so it should stay in for 3.7's docs (probably removed for 3.9's).